modin/core/io/text/text_file_dispatcher.py

```diff
@@ -619,7 +632,8 @@ def _launch_tasks(cls, splits: list, **partition_kwargs) -> Tuple[list, list, li
         for idx, (start, end) in enumerate(splits):
             partition_kwargs.update({"start": start, "end": end})
             *partition_ids[idx], index_ids[idx], dtypes_ids[idx] = cls.deploy(
-                func=cls.parse,
+                func=cls.get_parse_func(),
```

Comment:
We could preprocess the func outside the loop like we do in partitions.

Reply:
It doesn't need to be done since we are serializing and caching the result. Caching for the [entire ](https://github.com/modin-project/modin/pull/5678#discussion_r1131145293) workflow.